library/src/morpheus-master/sass/_defaults.scss

```diff
@@ -307,6 +307,7 @@ $tool-menu-collapse-hover-border-top:                      none !default;
 $tool-menu-collapse-hover-shadow:                          0 -1px 3px rgba(0,0,0,0.3) !default;
 $tool-menu-collapse-active-text-color:                     #FFFFFF !default;
 $tool-menu-collapse-active-background-color:               #15597e !default;
+$tool-menu-collapse-active-icon-color:                     #FFFFFF !default;
```

Comment:
I did it that way since in other places there is both a text color and an icon color, even when no text will be shown by default. Can do, I was just trying to stick to convention.